src/routes/Main.js

```diff
@@ -94,6 +95,16 @@ class Main extends Component {
       },
       records: 'configs',
     },
+    createInventory: {
```

Comment:
Are you sure we need this on the list? Maybe it should be in POLineForm only, since it's used only there